### ISSUE TITLE
Visual focus indicator for home agency select is low contrast

### ISSUE THREAD

**Issue by cmajel** [cal-itp/benefits#3264]
The new agency selector on the homepage has a visual focus indicator that maps to #045b86 / `var(--primary-color)`. 

That is a divergent pattern from the standard focus pattern shown below, which maps to #fdb714/ `var(--focus-color)`, and doesn't meet a [3:1 contrast ratio per WCAG](https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance.html). 

## To Reproduce

1. [Focus on home agency select in the test application ](https://test-benefits.calitp.org/)

## Expected behavior

Update the focus indicator for the home agency select to the standard pattern

## Screenshots

<img width="1261" height="1183" alt="Image" src="https://github.com/user-attachments/assets/592798e5-1faf-43dc-ba5c-278142d8d6b7" />
<img width="1261" height="1183" alt="Image" src="https://github.com/user-attachments/assets/149b2d38-5e00-41c5-a76d-4899dc699546" />

## Additional context

This also revealed the default #fdb714/ `var(--focus-color)`doesn't meet a 3:1 ratio of white backgrounds, so I'll open a separate issue to evaluate this holistically. This change will fix a one-off contrast issue on a high traffic page (home)
